index.html

```diff
@@ -1479,6 +1479,88 @@ <h3>Types</h3>
 
       </section>
 
+      <section>
+        <h3>Name</h3>
+
+        <p>
+This specification defines the <code>name</code> <a>property</a> for
+expressing the human readable name for a <a>credential</a>.
+        </p>
+
+        <dl>
+          <dt><var>name</var></dt>
+          <dd>
+A <a>credential</a> MAY have a <code>name</code> <a>property</a>. The
+value of the <code>name</code> <a>property</a> MUST be a string value 
+representing the human readable name for the <a>credential</a>
+          </dd>
+        </dl>
+
+        <pre class="example nohighlight" title="Usage of name property">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v2",
+    "https://www.w3.org/2018/credentials/examples/v1"
```

Comment:
I believe that `https://www.w3.org/2018/credentials/examples/v1` defines/redefines `name` -- will need to see if this example still works with a v2 context.

Reply:
it uses schema.org name: https://github.com/w3c/vc-data-model/blob/master/contexts/credentials/examples/v1#L36

Reply:
Yeah need to test but it should be ok as they have the same iri?